### ISSUE TITLE
Dynamically refresh device list

### ISSUE THREAD

**Issue by jacksongoode** [flipswitchingmonkey/FlexASIO_GUI#17]
Currently one has to close and reopen the app to see the new input/outputs. It would be nice to have the program refresh the device list when a new one is detected by the system - or just allow a user to refresh the list manually with a button or Ctrl+R.

**Comment by flipswitchingmonkey** [comment]
Will add manual refresh, there's no automatic notification currently, the GUI would have to constantly poll the devices and this just feels like a waste of cycles...

**Comment by jacksongoode** [comment]
Makes sense. I wasn't sure if there was a way of receiving a trigger to poll when Windows detects a change in devices.

**Comment by jacksongoode** [comment]
@flipswitchingmonkey I've been testing the refresh feature with various devices and it doesn't seem to work with any devices I've connected.
